contrib/freshmaker-cli

```diff
@@ -18,14 +18,21 @@ Example usage:
         ./freshmaker-cli --redhat --stage --name httpd24-docker
     - Search for events in particular state
         ./freshmaker-cli --redhat --stage --state skipped
+    - Search for RHSA advisories from ET ignored by Freshmaker from 5 days before to current date
+       REQUESTS_CA_BUNDLE=/etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt ./freshmaker-cli --redhat  --errata --days-ago 5
+       (REQUESTS_CA_BUNDLE should be passed to avoid SSL errors)
```

Comment:
This only applies if the requests library was installed via pip and not via a system package manager such as dnf. I would remove mentioning this environment variable since it's system specific.